src/Form/Control/Lookup.php

```diff
@@ -251,10 +251,10 @@ protected function initQuickNewRecord()
 
         $vp->set(function ($page) {
             $form = \Atk4\Ui\Form::addTo($page);
-
-            $entity = $this->model->createEntity();
-
-            $form->setModel($entity->setOnlyFields($this->plus['fields'] ?? []));
+            
```

Comment:
trailing spaces need to be removed, see CS diff

Reply:
@mvorisek See new commit